lib/sles4sap.pm

```diff
@@ -1489,6 +1569,9 @@ sub post_fail_hook {
     # NW installation logs, if needed
     $self->upload_nw_install_log if get_var('NW');
 
+    # ASE installation logs, if needed
+    $self->upload_ase_logs if $self->IS_ASE;
```

Comment:
an `if` on `IS_ASE` is both here and inside the called function

Reply:
Yes.